components/count-badge/README.md

```diff
@@ -25,6 +25,17 @@ The `d2l-count-badge` element is a web component to display a number count, eith
 | `max-digits`, default: `2` when `type="notification"` | Number | Optionally specify a digit limit, after which numbers are truncated. Defaults to two for `"notification"` type and no limit for `"count"` type.
 | `hide-zero`, default: `false` | Boolean | Optionally choose not to show the count badge when the number is zero. |
 
+### Accessibility Properties
+
+To make your `d2l-count-badge` accessible, use the following properties when applicable:
+
+| Attribute | Description |
+|--|--|
+| `description`, required | A description of the badge for screen-readers, to be used as an aria-label. NOTE: Only the description will be read by screen-readers (not the number). |
+| `tab-stop` | Optionally choose to make the badge a tab stop. |
+| `announce-changes` | Optionally choose to announce changes to the badge with an aria-live region. If the number property is changed, the description will be read by screenreaders. |

```

Comment:
I think we typically still include accessibility-related properties in the property list above, and then just call them out again in these lists with extra guidance. Although I'm not sure if @margaree has thoughts here or if we've changed our approach with the Daylight site?

Reply:
Yup that's still correct